sql-cloudant/README.md

```diff
@@ -62,6 +62,8 @@ cloudant.protocol|https|protocol to use to transfer data: http or https
 cloudant.host||cloudant host url
 cloudant.username||cloudant userid
 cloudant.password||cloudant password
+cloudant.useQuery|false|When enabled, for query not using index or view, _find will be used instead of _all_docs, some query predicates will be driven into datastore. However, RDD partition is ONE during _find, so parallel loading is not achieved
```

Comment:
Ok - I'm still having a hard time understanding the statement `some of the query predicates will be driven into datastore`.  What do you mean by driven? Does it have to do with `partitions used to drive JsonStoreRDD loading query result in parallel` ?
Also, `As RDD partition is ONE during _find` - one what? set to one partition for the `jsonstore.rdd.partitions` configuration? 
And for `parallel loading is not achieved` - This sounds awkward to me.  Is it not achieved because the `_find` endpoint only supports one partition?